src/__tests__/AppUX.test.js

```diff
@@ -0,0 +1,58 @@
+import { fireEvent, render, within } from '@testing-library/react';
+import fetchMock from 'fetch-mock';
+import { Provider } from 'react-redux';
+import App from '../App';
+import testStore from '../redux/testStore';
+import getMockCovidData from '../__mocks__/getMockCovidData';
+
+global.scrollTo = jest.fn();
+
+const clickElements = (elements, indexes = []) => {
+  indexes.forEach((index) => {
+    fireEvent.click(elements[index]);
+  });
+};
+
+let tree;
+describe('testing UX for the whole App', () => {
+  const url = 'https://api.covid19tracking.narrativa.com/api/';
+  fetchMock.get(url, getMockCovidData());
+
+  beforeEach(() => {
+    fetchMock.reset();
+    tree = render(
+      <Provider store={testStore}>
+        <App />
+      </Provider>,
+    );
+  });
+
+  test('testing homepage UX', async () => {
+    const countries = await tree.findAllByTestId('country');
+    expect(countries).toHaveLength(195);
+    const input = await tree.findByRole('textbox');
+    fireEvent.change(input, { target: { value: 'Colombia' } });
+    const country = await tree.findByTestId('country');
+    expect(await within(country).findByText('Colombia')).toBeTruthy();
+    expect(await tree.queryByTestId('backBtn')).toBeFalsy();
+  });
```

Comment:
- [ ] According to the image below, this test seems to be failing, please fix it.

![dico-test](https://user-images.githubusercontent.com/36542986/142294499-40a3b26d-8415-4f56-b089-7a398c5927da.jpg)